Hi3Helper.Core/packages.lock.json

```diff
@@ -4,8 +4,8 @@
     "net8.0": {
       "Google.Protobuf": {
         "type": "Transitive",
-        "resolved": "3.27.2",
-        "contentHash": "0wdgA3LO9mBS477jieBFs4pU1sWhVtwv/P+i9nAEiFDQyUA7PPHDBbJL1CeqYtV18jLiq9og4n7wSVCO171OBg=="
+        "resolved": "3.27.3",
```

Comment:
Its fine to up this to 3.28.0

Reply:
I will update the NuGet on Hi3Helper.Core once this PR is merged